src/frontend/src/optimizer/plan_node/stream_simple_agg.rs

```diff
@@ -27,38 +27,37 @@ use crate::stream_fragmenter::BuildFragmentGraphState;
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct StreamSimpleAgg {
     pub base: PlanBase,
-    logical: generic::Agg<PlanRef>,
+    core: generic::Agg<PlanRef>,
 
     /// The index of `count(*)` in `agg_calls`.
     row_count_idx: usize,
 }
 
 impl StreamSimpleAgg {
-    pub fn new(logical: generic::Agg<PlanRef>, row_count_idx: usize) -> Self {
-        assert_eq!(logical.agg_calls[row_count_idx], PlanAggCall::count_star());
+    pub fn new(core: generic::Agg<PlanRef>, row_count_idx: usize) -> Self {
+        assert_eq!(core.agg_calls[row_count_idx], PlanAggCall::count_star());
 
-        let input = logical.input.clone();
+        let input = core.input.clone();
         let input_dist = input.distribution();
         let dist = match input_dist {
             Distribution::Single => Distribution::Single,
             _ => panic!(),
         };
 
         // Empty because watermark column(s) must be in group key and simple agg have no group key.
-        let watermark_columns = FixedBitSet::with_capacity(logical.output_len());
+        let watermark_columns = FixedBitSet::with_capacity(core.output_len());
 
         // Simple agg executor might change the append-only behavior of the stream.
-        let base =
-            PlanBase::new_stream_with_logical(&logical, dist, false, false, watermark_columns);
+        let base = PlanBase::new_stream_with_logical(&core, dist, false, false, watermark_columns);
```

Comment:
Why `from_logical` for batch while `with_logical` for stream?

Reply:
Fixed in [53b361c](https://github.com/risingwavelabs/risingwave/pull/12980/commits/53b361ccdfb189898f044cbb38e902e1fb3c6b32)